grpc/lookup/v1/rls_config.proto

```diff
@@ -179,11 +179,9 @@ message RouteLookupConfig {
   // treated the same as an RPC error from the RLS.
   repeated string valid_targets = 8;
 
-  // This value provides a default target to use if needed.  It will be used for
-  // request processing strategy SYNC_LOOKUP_DEFAULT_TARGET_ON_ERROR if RLS
-  // returns an error, or strategy ASYNC_LOOKUP_DEFAULT_TARGET_ON_MISS if RLS
-  // returns an error or there is a cache miss in the client.  It will also be
-  // used if there are no healthy backends for an RLS target.  Note that
+  // This value provides a default target to use if needed.  If nonempty
+  // (implies request processing strategy SYNC_LOOKUP_DEFAULT_TARGET_ON_ERROR
```

Comment:
Maybe specify the behavior if `default_target` is nonempty but `request_processing_strategy` is not SYNC_LOOKUP_DEFAULT_TARGET_ON_ERROR.

Reply:
In the next couple of weeks I am planning to remove request_processing_strategy so I'd say let's not worry about the exact semantics of this corner case.

Reply:
sg